buildtest/cli/config.py

```diff
@@ -23,6 +23,9 @@ def config_cmd(args, configuration, editor, system):
     if args.config in ["view", "v"]:
         view_configuration(configuration, theme=args.theme, pager=args.pager)
 
+    elif args.config in ["profile"]:
+        view_profile(configuration, theme=args.theme, pager=args.pager)
+
```

Comment:
Here it is saying that the `view_profile` method should be called when someone write the `profile` argument in the terminal. However, you have not written any of this logic in the __init__.py file. As I stated on a previous comment, you need to add this somewhere in the config_menu method.